specs/experimental/standard-l2-genesis.md

```diff
@@ -60,30 +62,31 @@ configurability.
+| `SEQUENCER_FEE_VAULT_CONFIG`        | `uint8(3)` | Sets the Fee Vault Config for the `SequencerFeeVault` |
+| `L1_CROSS_DOMAIN_MESSENGER_ADDRESS` | `uint8(4)` | Sets the `L1CrossDomainMessenger` address             |
+| `L1_ERC_721_BRIDGE_ADDRESS`         | `uint8(5)` | Sets the `L1ERC721Bridge` address                     |
+| `L1_STANDARD_BRIDGE_ADDRESS`        | `uint8(6)` | Sets the `L1StandardBridge` address                   |
+| `REMOTE_CHAIN_ID`                   | `uint8(7)` | Sets the chain id of the base chain                   |
 
 ## `SystemConfig`
 
 ### `ConfigUpdate`
 
 The following `ConfigUpdate` event is defined where the `CONFIG_VERSION` is `uint256(0)`:
 
-| Name | Value | Definition | Usage |
-| ---- | ----- | --- | -- |
-| `BATCHER` | `uint8(0)` | `abi.encode(address)` | Modifies the account that is authorized to progress the safe chain |
-| `FEE_SCALARS` | `uint8(1)` | `(uint256(0x01) << 248) \| (uint256(_blobbasefeeScalar) << 32) \| _basefeeScalar` | Modifies the fee scalars |
-| `GAS_LIMIT` | `uint8(2)` | `abi.encode(uint64 _gasLimit)` | Modifies the L2 gas limit |
-| `UNSAFE_BLOCK_SIGNER` | `uint8(3)` | `abi.encode(address)` | Modifies the account that is authorized to progress the unsafe chain |
-| `EIP_1559_PARAMS` | `uint8(4)` | `uint256(uint64(uint32(_denominator))) << 32 \| uint64(uint32(_elasticity))` | Modifies the EIP-1559 denominator and elasticity |
+| Name                  | Value      | Definition                                                                        | Usage                                                                |
+| --------------------- | ---------- | --------------------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| `BATCHER`             | `uint8(0)` | `abi.encode(address)`                                                             | Modifies the account that is authorized to progress the safe chain   |
+| `FEE_SCALARS`         | `uint8(1)` | `(uint256(0x01) << 248) \| (uint256(_blobbasefeeScalar) << 32) \| _basefeeScalar` | Modifies the fee scalars                                             |
+| `GAS_LIMIT`           | `uint8(2)` | `abi.encode(uint64 _gasLimit)`                                                    | Modifies the L2 gas limit                                            |
+| `UNSAFE_BLOCK_SIGNER` | `uint8(3)` | `abi.encode(address)`                                                             | Modifies the account that is authorized to progress the unsafe chain |
+| `EIP_1559_PARAMS`     | `uint8(4)` | `uint256(uint64(uint32(_denominator))) << 32 \| uint64(uint32(_elasticity))`      | Modifies the EIP-1559 denominator and elasticity                     |
+| `FEE_VAULT_ADMIN`     | `uint8(5)` | `abi.encode(address)`                                                             | Modifies the fee vault admin                                         |
```

Comment:
This doesn't seem right, why is fee vault admin added here? These are generally values that go into L1 attributes tx and we dont want to put the fee vault admin in there. The `ConfigUpdate` events influence L2 consensus